### PR TITLE
Getting started: Move Dashboard explainer

### PR DESCRIPTION
**Pull request by c0ffeeca7**
- from Terminology to Dashboard section

## Proposed change

Getting started: Move Dashboard explainer

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/getting-started/onboarding_dashboard.markdown [hunk 1]
```diff
@@ -46,6 +52,6 @@ The procedure below is optional. The idea is to learn some basics on changing th
    - On the bottom of the card, select the number or use the minus button to enter `1`.
    - Repeat this on other cards, if you want to move them around.
    - When you are done, in the top right corner, select **Done**.
-9. Congratulations! You have completed your first dashboard customization.
+9. Congratulations! You have completed your first dashboard customization dashboards.
```
Comment (klaasnicolaas) [inline source/getting-started/onboarding_dashboard.markdown]:
```
9. Congratulations! You have completed your first dashboard customization dashboards.
```
Is this a correct sentence? `dashboard customization dashboards.`...? 🤔

Comment (c0ffeeca7) [reply source/getting-started/onboarding_dashboard.markdown]: Thank you for the catch, @klaasnicolaas!

Comment (c0ffeeca7) [inline source/getting-started/onboarding_dashboard.markdown]:
```suggestion
9. Congratulations! You have completed your first dashboard customization.
```

**Review by klaasnicolaas — APPROVED** [review]
✅ LGTM

**Pull request merged**